dev.qwery.AddWater.json

```diff
@@ -0,0 +1,102 @@
+{
+    "id" : "dev.qwery.AddWater",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "addwater",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=~/.mozilla/firefox/:rw",
+        "--filesystem=~/.var/app/org.mozilla.firefox/.mozilla/firefox/:rw",
+        "--filesystem=~/.var/app/org.mozilla.firefox/.mozilla/Firefox/:rw",
```

Comment:
The second is doesn't seem to exist. it's `firefox` in lowercase.

Reply:
I was concerned they may change it in the future given Thunderbird's ID _is_ uppercase. But I'll remove it for now.